src/Components/Filters/Filters.jsx

```diff
@@ -56,7 +55,7 @@ export const Filters = ({ sliderValue, setSliderValue }) => {
               onChange={() =>
                 dispatch({
                   type: "toggleInventory",
-                  payload: "TOGGLE_INVENTORY"
+                  payload: "TOGGLE_INVENTORY",

```

Comment:
Please change the `dispatch` implementation. The values you're passing to `payload` should go in `type` and payload must be any `data` that you want to manipulate in the reducer function
```suggestion
                  type: "TOGGLE_INVENTORY"
```